phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/apps/AppsController.java

```diff
@@ -85,6 +97,38 @@ public void loadApps() {
         });
 
         this.view.setOnSelectCategory(categoryDTO -> this.view.populateApps(categoryDTO.getApplications(), settingsManager));
+
+        this.view.setOnSetDefaultCategoryIcons(categoryDTOS -> {
+            // set default category icons
+            try {
+                StringBuilder cssBuilder = new StringBuilder();
+                for (CategoryDTO category : categoryDTOS) {
+                    cssBuilder.append("#" + category.getName().toLowerCase() + "Button{\n");
+                    String categoryIcon = category.getIcon();
+                    if (StringUtils.isEmpty(categoryIcon) || ! new File(new URI(categoryIcon)).exists()) {
```

Comment:
The file will never exists if the icon is a remote file or if it is packaged inside a .jar fie